.github/workflows/changelogs.yml

```diff
@@ -65,22 +65,27 @@ jobs:
           echo "  - $file"
         done
         echo
-        echo "See https://github.com/uyuni-project/uyuni/wiki/Contributing for a guide to writing checklogs."
+        echo "See https://github.com/uyuni-project/uyuni/wiki/Contributing for a guide to writing changelogs."
         exit 1
-    - id: changelogs
-      name: Get modified changelog files
-      if: "!contains(github.event.pull_request.body, '[x] No changelog needed')"
-      uses: Ana06/get-changed-files@v2.2.0
-      with:
-        filter: '*.changes.*'
-    - name: Fail if no changelog entries are added
-      if: steps.changelogs.conclusion == 'success' && steps.changelogs.outputs.added_modified == ''
+    - name: Test changelog entries
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think this won't work on forks, as the credentials won't be present there, so we'll have the same problem we had for SonarQube? @cbosdo, am I right about this one?